src/content/peerconnection/webaudio-input/js/main.js

```diff
@@ -68,11 +68,11 @@ function gotStream(stream) {
     var filteredStream = webAudio.applyFilter(stream);
 
     var servers = null;
-
-    pc1 = new webkitRTCPeerConnection(servers); // jscs:ignore requireCapitalizedConstructors
+    /*eslint new-cap: 0*/
```

Comment:
I played around with this. 

The latest comment in the code seems to set the rule for the whole file: adding `/*eslint new-cap: 2*/` after the first declaration seems to cause both lines to get a warning. Not ideal...